src/Utilities/PaymentHelpers.res

```diff
@@ -1972,3 +1972,57 @@ let deletePaymentMethod = (~ephemeralKey, ~paymentMethodId, ~logger, ~customPodU
     JSON.Encode.null->resolve
   })
 }
+
+let calculateTax = (
+  ~apiKey,
+  ~paymentId,
+  ~clientSecret,
+  ~paymentMethodType,
+  ~shippingAddress,
+  ~logger,
+  ~customPodUri,
+) => {
+  open Promise
+  let endpoint = ApiEndpoint.getApiEndPoint()
+  let headers = [("Content-Type", "application/json"), ("api-key", apiKey)]
+  let uri = `${endpoint}/payments/${paymentId}/calculate_tax`
```

Comment:
Please add INIT log when the API is started. Confirm with @Sanskar2001 once what is the convention for that